MarketAnalyzer.Core/Library1.fs

```diff
@@ -19,10 +19,24 @@ module StockAnalysis =
         let url = sprintf "http://ichart.finance.yahoo.com/table.csv?s=%s&a=%i&b=%i&c=%i&d=%i&e=%i&f=%i&g=d&ignore=.csv" symbol (fromDate.Month - 1) (fromDate.Day) (fromDate.Year) (toDate.Month - 1) (toDate.Day) (toDate.Year)
         let stockRawData = CsvProvider<IgnoreErrors=true,Sample="Date (Date),Open (float),High (float),Low (float),Close (float),Volume (int64),AdjClose (float)">.Load url
         let stockRows = stockRawData.Rows
-        Series.ofObservations [ for d in stockRows -> (d.Date, d.Close)]
+        series [ for d in stockRows -> (d.Date, d.Close)]
```

Comment:
Thank you so much Tomas.


There are a couple more changes id like to add to this, now that this is in. 


Much appreciated.


Duncan






Sent from Windows Mail





From: Tomas Petricek
Sent: ‎Sunday‎, ‎25‎ ‎January‎ ‎2015 ‎8‎:‎00‎ ‎AM
To: duncanesson/MarketAnalyzer





In MarketAnalyzer.Core/Library1.fs:
> @@ -19,10 +19,24 @@ module StockAnalysis =
>          let url = sprintf "http://ichart.finance.yahoo.com/table.csv?s=%s&a=%i&b=%i&c=%i&d=%i&e=%i&f=%i&g=d&ignore=.csv" symbol (fromDate.Month - 1) (fromDate.Day) (fromDate.Year) (toDate.Month - 1) (toDate.Day) (toDate.Year)
>          let stockRawData = CsvProvider<IgnoreErrors=true,Sample="Date (Date),Open (float),High (float),Low (float),Close (float),Volume (int64),AdjClose (float)">.Load url
>          let stockRows = stockRawData.Rows
> -        Series.ofObservations [ for d in stockRows -> (d.Date, d.Close)]
> +        series [ for d in stockRows -> (d.Date, d.Close)]


The series function is just a shorter name for Series.ofObservations.

—
Reply to this email directly or view it on GitHub.